src/features/LoginView/GoogleLogin.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { StyleSheet, View, TouchableOpacity, Image } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import SafeArea from '../../components/SafeArea';
+import { theme } from '../../infrastructure/index';
+import { BiteShareContext } from '../../BiteShareContext';
+import * as Google from 'expo-google-app-auth';
+import googleImg from '../../../assets/google-signin.png';
+
+const GoogleLogin = () => {
+  const navigation = useNavigation();
+  const { state: { authenticated, nickname }, dispatch } = useContext(BiteShareContext);
+
+  const signInAsync = async () => {
+    console.log('GoogleLogin.js 6 | logging in');
+    try {
+      const { type, user } = await Google.logInAsync({
+        iosClientId: '90738707092-9ik3l59ad22hfghgrms9ti5cgn2da675.apps.googleusercontent.com',
+        // androidClientId: '<YOUR_ANDROID_CLIENT_ID>',
```

Comment:
Don't we need androidClientId? It doesn't work on my android.

Reply:
Yeah, I think I can set that up for Android. The location permissions are currently only configured to iOS as well. I wasn't sure if we should account for everything on android since we won't present it on android